release-notes/10.0/preview/preview3/api-diff/Microsoft.WindowsDesktop.App/10.0-preview3_System.Windows.Forms.Design.md

```diff
@@ -0,0 +1,28 @@
+# System.Windows.Forms.Design
+
+```diff
+  namespace System.ComponentModel.Design
+  {
+      public class DesignerActionMethodItem : System.ComponentModel.Design.DesignerActionItem
+      {
+-         public DesignerActionMethodItem(System.ComponentModel.Design.DesignerActionList actionList, string memberName, string displayName, bool includeAsDesignerVerb) : base(default(string?), default(string?), default(string?));
+-         public DesignerActionMethodItem(System.ComponentModel.Design.DesignerActionList actionList, string memberName, string displayName, string category, bool includeAsDesignerVerb) : base(default(string?), default(string?), default(string?));
+-         public DesignerActionMethodItem(System.ComponentModel.Design.DesignerActionList actionList, string memberName, string displayName, string category, string description, bool includeAsDesignerVerb) : base(default(string?), default(string?), default(string?));
+-         public DesignerActionMethodItem(System.ComponentModel.Design.DesignerActionList actionList, string memberName, string displayName, string category, string description) : base(default(string?), default(string?), default(string?));
+-         public DesignerActionMethodItem(System.ComponentModel.Design.DesignerActionList actionList, string memberName, string displayName, string category) : base(default(string?), default(string?), default(string?));
+-         public DesignerActionMethodItem(System.ComponentModel.Design.DesignerActionList actionList, string memberName, string displayName) : base(default(string?), default(string?), default(string?));
+-         public virtual bool IncludeAsDesignerVerb { get; }
++         public virtual bool? IncludeAsDesignerVerb { get; }
+-         public virtual string MemberName { get; }
++         public virtual string? MemberName { get; }
+-         public System.ComponentModel.IComponent RelatedComponent { get; set; }
++         public System.ComponentModel.IComponent? RelatedComponent { get; set; }
++         public DesignerActionMethodItem(System.ComponentModel.Design.DesignerActionList? actionList, string? memberName, string? displayName, bool? includeAsDesignerVerb) : base(default(string?), default(string?), default(string?));
```

Comment:
@Tanya-Solyanik let me know if this file doesn't look correct, the diff is unfortunately not showing each API before vs after.

Reply:
Curios - did the sort change?

Reply:
The APIs get sorted before and after, but I think the differ is grouping sequential differences as a single one. That's alright, as long as the actual changes displayed here are correct to you.